src/earn/EarnEnterAmount.tsx

```diff
@@ -669,6 +777,12 @@ const styles = StyleSheet.create({
     justifyContent: 'space-between',
     alignItems: 'flex-start',
   },
+  lineItem: {
```

Comment:
```suggestion
  bottomSheetLineItem: {
```
to remove ambiguity, or consider combining with `txDetailsLineItem`